server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -329,10 +334,18 @@ public void close() throws IOException {
 
     protected long getMinReferencedGen() throws IOException {
         assert readLock.isHeldByCurrentThread() || writeLock.isHeldByCurrentThread();
-        long minReferencedGen = Math.min(
+        long minSeqNo, minReferencedGen;
+        if (this.indexSettings.isRemoteStoreEnabled()) {
```

Comment:
`RemoteFsTranslog` is already remote and cannot be enabled independently why is this check needed

Reply:
Answered in previous comment.

Reply:
As discussed , this introduces different behavior in code . So we are removing this for now. Will take it along in https://github.com/opensearch-project/OpenSearch/issues/7581